litellm/types/utils.py

```diff
@@ -1315,6 +1315,7 @@ class ResponseFormatChunk(TypedDict, total=False):
     "num_retries",
     "context_window_fallback_dict",
     "retry_policy",
+    "retry_strategy",
```

Comment:
@krrishdholakia Can you take a look at https://github.com/BerriAI/litellm/pull/6520#issue-2625465142 and let me know what your thoughts are about how to best allow users to opt-in to retries with exponential backoff? We need this for robustness to rate limiting *and* transient network failures in DSPy.

If this direction makes sense, I can add parameter / docstring updates to `completions()`, etc.

Alternatively, perhaps we want to extend the Router `RetryPolicy` to support this instead? Or maybe we want both?

Let me know what your thoughts are here.

Reply:
cc @okhat

Reply:
+1